.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: Terraform CI
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'modules/**'
+  pull_request:
+    paths:
+      - 'modules/**'
+
+env:
+  TF_VERSION: 1.4.1
+  TF_IN_AUTOMATION: true
+  TF_PLUGIN_CACHE_DIR: ${{ github.workspace }}/.terraform.d/plugin-cache
+
+jobs:
+    terraform:
+        name: 'Terraform CI'
+        runs-on: ubuntu-latest
```

Comment:
We need to run it on Databricks-managed runners, like this:

https://github.com/databricks/databricks-jdbc-dev/blob/main/.github/workflows/prCheck.yml#L18

Reply:
Thx, but I get "Access forbidden from this IP address" when I follow the link. 
Any chance allowing running this ci job on gh hosted runners?

Reply:
I've updated the link

Reply:
Not sure if this would work for prs from non gh org members. I'm closing this one for now.